### DISCUSSION TITLE
Checking out the flow of github

### DISCUSSION THREAD

**Discussion by ahsansoftengineer** [community/community#20574] [Other Feature Feedback, Questions, & Ideas]
## Check list for intro of github

> Forking
> Give the Stars
> Contribute
> Pull Request
> etc...

**Comment by 4po** [comment]
I don't understand but okay !

**Comment by topstarwebking** [comment]
@ahsansoftengineer , What you mean?

**Reply by ahsansoftengineer** [reply]
Sorry for what
